packages/cli/commons/src/utils/ux.ts

```diff
@@ -0,0 +1,16 @@
+import {CliUx} from '@oclif/core';
+import {red, green} from 'chalk';
+
+export function starSpinner(task: string) {
+  if (CliUx.ux.action.running) {
+    CliUx.ux.action.stop(green('✔'));
+  }
+  CliUx.ux.action.start(task);
+}
+
+export function stopSpinner(err?: Error) {
```

Comment:
Not sure to understand, why a spinner can only finish with an error?

Reply:
Now that you mention it, I think the error message in the spinner is redundant. We can just have an boolean argument like
`stopSpinner(success = true)`